auditing/meilisearch.go

```diff
@@ -172,7 +175,13 @@ func (a *meiliAuditing) Search(filter EntryFilter) ([]Entry, error) {
 	if err != nil {
 		return nil, err
 	}
-	indexes, err := a.client.GetIndexes(&meilisearch.IndexesQuery{})
+	relevantIndexLimit := defaultRelevantIndexSearchLimit
+	if a.keep == 0 {
```

Comment:
Isn't this the `!= 0` case? As otherwise we set the limit to `1` while having a lot of indexes in the backend?

Not sure if I'd prefer a function for retrieving all indexes such that we can re-use it inside the index cleanup as well. Then it's equivalent in both places.

Reply:
Added the function, but made an adjustment. Now we fetch the expected amount of indexes. In many cases this should be fine.